docs/VAULT.md

```diff
@@ -0,0 +1,30 @@
+# Using Vault to keep tabs on environment variables
+
+## Prerequisites
+
+- An instance of [Hashcorps Vault](https://www.vaultproject.io/) running
+  - Using a docker-based approach is described [here](vault/setup-vault-using-docker.md)
+- A Token (or Username and Password) as credentials for accessing Vault
+
+## Put stuff into Vault
+
+Creating content in Vault can be done either using curl (not recommended) og using Vault as a client.
+A simple curl example can be found [here](vault/setup-vault-using-docker.md). Use a native vault application whenever possible. They are available for download [here](https://www.vaultproject.io/downloads.html). Using vault via docker requires you to be inside a running vault docker container. Otherwise the steps are the same.
+
+The below example assume that you have the following environment variables:
+
+- `$VAULT_TOKEN` containing the token to use as access credentials
+- `$VAULT_SERVER_URL` containing the URL to your vault server, i.e. `http://vault.domain.local:8200`
+
+If needed, enter the docker container: `docker exec -it vault /bin/sh`
```

Comment:
Maybe specify `library/` prefix, so as to ensure only the official Vault-image?
(If it is [this one](https://hub.docker.com/_/vault/) you were referring to).

AKA: `docker exec -it library/vault /bin/sh`

Reply:
fixed